warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/MultiRFileOutputFormatter.java

```diff
@@ -97,7 +91,7 @@
     protected String extension;
     protected Configuration conf;
     protected Map<String,ConfigurationCopy> tableConfigs;
-    protected Map<String,String> tableIds = null;
+    protected String[] tableIds = null;
```

Comment:
I see why I did it this way now.  I'd forgotten. IngestJob, partitioners, and PartitionerCache (maybe more?) accept the string array of tableIds as an argument to several methods (and have for a long time, it seems) which would cause me to touch more classes that I thought I should for this effort.  I can still update it if you want.